17.0 mig partner invoicing frequency (#42)
sygel-technology/sy-partner-contact: partner_invoicing_frequency/README.rst

.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
	:target: http://www.gnu.org/licenses/agpl
	:alt: License: AGPL-3

===================

> Invoicing Frecuency

Reviewer: ```suggestion
Invoicing Frequency
```

===================

> This module allows to assign a invoicing frecuency to contacts. This information is transferred to sale order, purchase orders, pickings and invoices, but it can be freely edited.

Reviewer: ```suggestion
This module allows to assign a invoicing frequency to contacts. This information is transferred to sale order, purchase orders, pickings and invoices, but it can be freely edited.
```

Installation
============

To install this module, you need to:

#. Only install

Configuration
=============

To configure the stages of the opportunities that can be transferred, you need to:

#. Go to Contact > Configuration > Invoicing Frecuencies

> #. Create as many invoicing frecuencies as needed.

Reviewer: ```suggestion
#. Create as many invoicing frequencies as needed.
```

Usage
=====

To use this module, you need to:

> #. Select a invoicing frecuency in a contact, that will be applied to ale order, purchase orders, pickings and invoices. It can be edited if necessary.

Reviewer: ```suggestion
#. Select a invoicing frecuency in a contact, that will be applied to sale orders, purchase orders, pickings and invoices. It can be edited if necessary.
```
